### ISSUE TITLE
Comments do not get placed after brace when converting to KRN style braces

### ISSUE THREAD

**Issue by shripada** [nicklockwood/SwiftFormat#4]
Consider this code that follows a  _"always braces in new line"_ style  with a comment following an if condition- 

```
var networkEnabled = true
if networkEnabled  //We want to enable the UI..
{
   
}
```
The tool ends up converting the above code to-
```
var networkEnabled = true
if networkEnabled  //We want to enable the UI.. {

}

```
This will result in compiler error as obviously the opening brace after the **if** is effectively commented out.

**Comment by nicklockwood** [comment]
Ooh, nice edge case :-) Thanks!

**Comment by nicklockwood** [comment]
Fixed in v0.3. Thanks!

**Comment by shripada** [comment]
Thank you @nicklockwood, now time to use it live in my project.
